src/main/java/com/google/sps/servlets/EventsServlet.java

```diff
@@ -0,0 +1,116 @@
+package com.google.sps.servlets;
+
+import com.google.api.client.util.DateTime;
+import com.google.api.services.calendar.Calendar;
+import com.google.api.services.calendar.model.Event;
+import com.google.api.services.calendar.model.Events;
+import com.google.api.services.calendar.model.EventDateTime;
+import com.google.common.base.Strings;
+import com.google.gson.Gson;
+import java.io.File;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Servlet for obtaining the Calendar ID of a group and adding or obtaining events
+ */
+@WebServlet("/events")
+public class EventsServlet extends AbstractEventsServlet {
+  private static final int MAX_EVENTS = 50;
+  private static final String CALENDAR_ID_PARAM = "calendarId";
+
+  private static final Gson gson = new Gson();
+
+  /**
+   * Obtain a list of events from the specified group if given the Calendar ID.
+   * Obtain the Calendar ID of a group given the Group ID. If both provided, return list of events.
+   */
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String calendarId = getParameter(request, "calendarid", "");
+    String groupId = getParameter(request, "groupid", "");
```

Comment:
Are these values going to be received in a query string?

Reply:
Yes, added a comment to clarify this